packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -450,7 +450,7 @@ export class PerformanceEvent {
     }
 
     public cancel(props?: ITelemetryProperties, error?: any): void {
-        if (this.markers.cancel !== undefined) {
+        if (this.markers.cancel) {
```

Comment:
can you also make cancel a bool on IPerformanceEventMarkers

Reply:
Actually it is to give choice to user to make cancel event go in error table.

Reply:
that's pretty unintuitive. how did it work before?

Reply:
Actually original code is correct (sorry, long hours, I already forgot what I did yesterday :).
Yes, the intention is to give a chose to user to use undefined | category of cancel event.

Reply:
It was unconditional, and it would always generate performance category event. There are places where we want to bring attention to cancelation, so want to use "error" category. The unfortunate side-effect of using different tables for different categories is that it might be harder to understand what happened if you only look at perf table (I wish all data was in one table), but I usually query all tables to see the flow of events.

Reply:
ok. sounds good. easy to iterate here anyway